builders/build/substrate-api/chopsticks.md

```diff
@@ -112,7 +111,7 @@ The `npx @acala-network/chopsticks dev` command forks a chain, and includes foll
 
 When running a fork, by default it will be accessible at `ws://localhost:8000`. You will be able to interact with the parachain via libraries such as [Polkadot.js](https://github.com/polkadot-js/common){target=_blank} and its [user interface, Polkadot.js Apps](https://github.com/polkadot-js/apps){target=_blank}.  
```

Comment:
break ws://localhost:8000 to a new line and wrap it in three backticks so its easy to copy